kubeflow/trainer/rhai/traininghub_test.py

```
@@ -690,5 +690,127 @@ def test_instrumentation_wrapper_flush_all_prints():
+    exec(wrapper, exec_globals)
+
+    # Should not raise exception when no files exist
+    try:
+        exec_globals["apply_progression_tracking"]()
+    except Exception as e:
+        pytest.fail(f"Cleanup should handle missing files gracefully, but raised: {e}")
+
+    print("test execution complete")
+
+
+def test_instrumentation_cleanup_continues_on_error(tmp_path):
```

Ruff (0.14.14)

[warning] Unused function argument: `tmp_path`

(ARG001)